x/ccv/provider/keeper/relay.go

```diff
@@ -213,57 +216,68 @@ func (k Keeper) EndBlockCIS(ctx sdk.Context) {
 	k.SetValsetUpdateBlockHeight(ctx, valUpdateID, uint64(ctx.BlockHeight()+1))
 }
 
-// OnRecvSlashPacket slashes and jails the given validator in the packet data
+// OnRecvSlashPacket receives a slash packet, validates it, then handles it if valid.
 func (k Keeper) OnRecvSlashPacket(ctx sdk.Context, packet channeltypes.Packet, data ccv.SlashPacketData) exported.Acknowledgement {
-	// check that the channel is established
+
+	// check that the channel is established, panic if not
 	chainID, found := k.GetChannelToChain(ctx, packet.DestinationChannel)
 	if !found {
 		// SlashPacket packet was sent on a channel different than any of the established CCV channels;
 		// this should never happen
 		panic(fmt.Errorf("SlashPacket received on unknown channel %s", packet.DestinationChannel))
```

Comment:
I think we should return an err ack here.

Reply:
Maybe we can add a comment to #568, but I'd like to avoid functionality changes in this PR as much as possible